src/rgw/rgw_user.cc

```diff
@@ -270,10 +270,10 @@ int rgw_get_user_info_from_index(RGWRados *store, string& key, rgw_bucket& bucke
 int rgw_get_user_info_by_uid(RGWRados *store,
                              const rgw_user& uid,
                              RGWUserInfo& info,
-                             RGWObjVersionTracker *objv_tracker,
-                             real_time *pmtime,
-                             rgw_cache_entry_info *cache_info,
-                             map<string, bufferlist> *pattrs)
+                             RGWObjVersionTracker * const objv_tracker,
```

Comment:
I think that we should gradually move '*' to the type it modifies, as unlike C, that's the style in Stroustrup.  We seem to follow that model for reference but not pointer types here.  I'm not going to put this remark everywhere ;).